### PR TITLE
Add github action to update the release.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: .github/workflows/update_release.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        run: |
+          SHA256=$(sha256sum $ARTIFACT | cut -d " " -f1)
+          sed -i "s/sha256 \".*/sha256 \"$SHA256\"/" Formula/jaguar.rb
+          sed -i "s,url \"https:/.*,url \"$URL\"," Formula/jaguar.rb
+          git diff
+
+      - name: Push
+        run: |
+          git config --global user.name 'Leon Gungadin Mogensen'
+          git config --global user.email 'LeonGungadinMogensen@users.noreply.github.com'
```
Comment (kasperl) [inline .github/workflows/update_release.yml]: leon@toit.io?

File: .github/workflows/update_release.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+name: update_release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The version of the release"
+        required: true
+        type: string
+
```
Comment (kasperl) [inline .github/workflows/update_release.yml:10]: Drop one newline.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: .github/workflows/update_release.yml (hunk 2 above)
Comment (floitsch) [reply .github/workflows/update_release.yml:10]: done.

File: .github/workflows/update_release.yml (hunk 1 above)
Comment (floitsch) [reply .github/workflows/update_release.yml]: done.

**Pull request merged**